### DISCUSSION TITLE
GlobaLeaks utilizza Cookies?

### DISCUSSION THREAD

**Discussion by rlovisetto** [globaleaks/globaleaks-whistleblowing-software#3653] [Q&A] [answered]
Un nostro cliente ci ha sottoposto la questione dei cookies. Ci chiede se la piattaforma utilizza cookies e informazioni su come eventualmente andarli a riportare nella cookies policy,
Dalla documentazione disponibile ho letto il ticket #730 e mi è sembrato di capire se sia stato rimosso l'utilizzo dei cookies. Non sono sicuro però che effettivamente non siano utilizzati i cookies.
Mi potreste ragguagliere su questo tema al fine di conformare la policy?

**Comment by evilaliv3** [comment, marked as answer]
GlobaLeaks non utilizza alcun tipo di Cookie.

Le sessioni per ragioni di sicurezza e privacy sono temporanee e vivono nel browser in cui sono create e sono implementate tramite normali headers HTTP e Session storage temporaneo. Ogni informazione di sessione viene persa alla chiusura del tab o del browser in cui l'applicazione è aperta.

**Comment by rlovisetto** [comment]
Grazie Giovanni
Tempestivo ed esauriente 😊

Da: Giovanni Pellerano ***@***.***>
Inviato: sabato 23 settembre 2023 18:23
A: globaleaks/GlobaLeaks ***@***.***>
Cc: Roberto Lovisetto - Conformis in Finance ***@***.***>; Author ***@***.***>
Oggetto: Re: [globaleaks/GlobaLeaks] GlobaLeaks utilizza Cookies? (Discussion #3653)


GlobaLeaks non utilizza alcun tipo di Cookie.

Le sessioni per ragioni di sicurezza e privacy sono temporanee e vivono nel browser in cui sono create e sono implementate tramite normali headers HTTP e Session storage temporaneo. Ogni informazione di sessione viene persa alla chiusura del tab o del browser in cui l'applicazione è aperta.
